src/lib/focusedWindowIsMaximised.mjs

```diff
@@ -0,0 +1,36 @@
+import getFocusedScreenInfo from "./getFocusedScreenInfo.mjs";
+import getMenuBarHeight from "./getMenuBarHeight.mjs";
+import { stageManagerMarginWidth } from "./config.mjs";
+import getFocusedWindowInfo from "./getFocusedWindowInfo.mjs";
+
+export default async function focusedWindowIsMaximised() {
+  const { focusedScreenX, focusedScreenY, focusedScreenWidth, focusedScreenHeight } =
+    await getFocusedScreenInfo();
+  const { focusedWindowX, focusedWindowY, focusedWindowWidth, focusedWindowHeight } =
+    await getFocusedWindowInfo();
+  const menuBarHeight = getMenuBarHeight(focusedScreenWidth, focusedScreenHeight);
+
+  const isInUpperLeftCorner =
```

Comment:
This does not work, unfortunately – the variables always are set to the coordinates and dimensions of the screen that currently is the main screen, i.e. that has the menu bar on it. 

I need to get the coordinates and dimensions of the screen that the currently focused window is on.

[See discussion on BTT forum](https://community.folivora.ai/t/get-width-and-height-of-the-display-that-the-currently-active-window-is-on-with-a-multi-monitor-setup/30234/2)